ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -155,7 +155,7 @@ def angle(
     deg: Optional[bool] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.angle(input)
```

Comment:
deg argument is missing

Reply:
deg is not implemented in paddle.degree function. It only require a one input.  
https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/angle_en.html